bin/storm.py

```diff
@@ -156,49 +95,92 @@ def get_classpath(extrajars, daemon=True, client=False):
     ret.extend(get_wildcard_dir(os.path.join(STORM_DIR, "extlib")))
     if daemon:
         ret.extend(get_wildcard_dir(os.path.join(STORM_DIR, "extlib-daemon")))
-    if STORM_EXT_CLASSPATH != None:
+    if STORM_EXT_CLASSPATH:
         ret.append(STORM_EXT_CLASSPATH)
-    if daemon and STORM_EXT_CLASSPATH_DAEMON != None:
+    if daemon and STORM_EXT_CLASSPATH_DAEMON:
         ret.append(STORM_EXT_CLASSPATH_DAEMON)
     ret.extend(extrajars)
-    return normclasspath(os.pathsep.join(ret))
+    return NORMAL_CLASS_PATH(os.pathsep.join(ret))
 
-def confvalue(name, extrapaths, daemon=True):
-    global CONFFILE
-    command = [
-        JAVA_CMD, "-client", get_config_opts(), "-Dstorm.conf.file=" + CONFFILE,
-        "-cp", get_classpath(extrapaths, daemon), "org.apache.storm.command.ConfigValue", name
-    ]
-    p = sub.Popen(command, stdout=sub.PIPE)
-    output, errors = p.communicate()
-    # python 3
-    if not isinstance(output, str):
```

Comment:
Is this check not necessary for Python 3 anymore?

Reply:
Has this question been addressed?

Reply:
Yes this has been resolved already by keeping the needed checks.